sql/core/benchmarks/WideTableBenchmark-results.txt

```diff
@@ -0,0 +1,17 @@
+================================================================================================
+projection on wide table
+================================================================================================
+
+OpenJDK 64-Bit Server VM 1.8.0_191-b12 on Linux 3.10.0-862.3.2.el7.x86_64
+Intel(R) Xeon(R) CPU E5-2670 v2 @ 2.50GHz
+projection on wide table:                Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
+------------------------------------------------------------------------------------------------
+split threshold 10                          38932 / 39307          0.0       37128.1       1.0X
+split threshold 100                         31991 / 32556          0.0       30508.8       1.2X
+split threshold 1024                        10993 / 11041          0.1       10483.5       3.5X
+split threshold 2048                          8959 / 8998          0.1        8543.8       4.3X
```

Comment:
@dongjoon-hyun In my mac, at most case, `2048` is the best.

Reply:
I tested it with openjdk (in one Linux VM), `2048` is also the best.

```
================================================================================================
projection on wide table
================================================================================================

OpenJDK 64-Bit Server VM 1.8.0_171-b10 on Linux 3.10.0-693.11.1.el7.x86_64
Intel Core Processor (Haswell)
projection on wide table:                Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
------------------------------------------------------------------------------------------------
split threshold 10                          23995 / 25673          0.0       22883.7       1.0X
split threshold 100                         12881 / 13419          0.1       12284.3       1.9X
split threshold 1024                          6435 / 7402          0.2        6137.2       3.7X
split threshold 2048                          5861 / 6766          0.2        5589.2       4.1X
split threshold 4096                          6464 / 7825          0.2        6164.6       3.7X
split threshold 8192                          7886 / 8742          0.1        7520.7       3.0X
split threshold 65536                       46143 / 48029          0.0       44005.6       0.5X
```

Reply:
@yucai . I trust you. :) Don't worry about that. The scope of this PR is not for choosing the best option.